lib/src/executable/options.dart

```diff
@@ -91,6 +91,18 @@ final class ExecutableOptions {
               "Stylesheets imported through load paths count as dependencies.")
       ..addFlag('verbose',
           help: "Print all deprecation warnings even when they're repetitive.")
+      ..addMultiOption('silence-deprecation',
+          help: 'Deprecations to ignore.',
+          allowedHelp: {
+            for (var deprecation in Deprecation.values)
+              if (deprecation
+                  case Deprecation(
+                    deprecatedIn: Version(),
+                    obsoleteIn: null,
+                    :var description?
+                  ))
+                deprecation.id: description,
```

Comment:
Between this and `--fatal-deprecation`, nearly half the length of the `--help` output is taken up by deprecation descriptions. WDYT about just linking to the Sass site's CLI docs instead?

Reply:
Yeah, I think that makes more sense. Is there a short version of https://sass-lang.com/documentation/cli/dart-sass so we can link to it with `#fatal-deprecation`, `#silence-deprecation` and `#future-deprecation`?